src/WebJobs.Script/Rpc/LanguageWorkerChannelManager.cs

```diff
@@ -35,9 +36,11 @@ public class LanguageWorkerChannelManager : ILanguageWorkerChannelManager
         private string _workerRuntime;
         private Action _shutdownStandbyWorkerChannels;
         private IDictionary<string, ILanguageWorkerChannel> _workerChannels = new Dictionary<string, ILanguageWorkerChannel>();
+        private IOptions<ManagedDependencyOptions> _managedDependencyOptions;
```

Comment:
Remove _managedDependencyOptions. LanguageWorkerChannelManager should not depend on JobHost level options